src/constants.ts

```diff
@@ -40,6 +40,12 @@ export enum SUPPORT_LANGUAGES {
   EN = 'en',
 }
 
+export const supportedLanguages = [
```

Comment:
`export const supportedLanguages = Object.values(SUPPORT_LANGUAGES);` is shorter and safer too if SUPPORT_LANGUAGES are updated?

Reply:
Also, there is dozens of `Object.values(SUPPORT_LANGUAGES)` in the codebase, they could be replaced with `supportedLanguages`

Reply:
Yeah should use object.values